webapp/src/components/Profile.tsx

```diff
@@ -1,7 +1,122 @@
+  Grid,
+  Avatar,
+  Tooltip,
+  Divider,
+  IconButton,
+  Typography,
+} from '@mui/material';
+
+import GitHubIcon from '@mui/icons-material/GitHub';
+import LinkedInIcon from '@mui/icons-material/LinkedIn';
+import LanguageIcon from '@mui/icons-material/Language';
+import EmailIcon from '@mui/icons-material/Email';
+
+const user = {
+  name: 'Matthew Morenez',
+  email: 'profile@example.com',
+  avatar:
+    'https://media.volinspire.com/images/95/e4/99/95e499b759ba57975a61c7bf66a3414dd5a2625e_profile.jpg',
+  github: 'https://github.com',
+  website: 'https://example.com',
+  summary:
+    'Lorem ipsum dolor, sit amet consectetur adipisicing elit. Officia, sit impedit? Cupiditate veniam eaque suscipit eligendi. Sint delectus enim earum non repellendus nihil numquam libero odit temporibus et, natus eaque?',
+  linkedIn: 'https://linkedin.com',
+};
 
 const Profile = () => {
-  return <div>Profile</div>;
+  return (
+    <Container fixed>
+      <Grid
+        container
+        sx={{
+          justifyContent: 'center',
+          alignItems: 'center',
+          display: 'flex',
+        }}
+        spacing={2}
+      >
+        <Grid
+          item
+          xs={12}
+          md={4}
+          sx={{
+            justifyContent: 'center',
+            alignItems: 'center',
+            display: 'flex',
+          }}
+        >
+          <Avatar
+            sx={{
+              width: 150,
+              height: 150,
+              border: '3px solid #fff',
+              outline: '2px solid #1976d2',
+            }}
+            src={user.avatar}
+          ></Avatar>
+        </Grid>
+        <Grid
+          item
+          xs={12}
+          md={6}
+          alignItems={{ md: 'flex-start', sm: 'center' }}
+          display="flex"
+          flexDirection="column"
+        >
+          <Typography component="h2" variant="h4">
+            {user.name}&apos;s Profile
+          </Typography>
+          <Typography
+            component="p"
+            sx={{ display: 'flex', alignItems: 'center', mt: 1 }}
+          >
+            <EmailIcon sx={{ mr: 1 }} color="primary" />
+            {user.email}
+          </Typography>
+          <Grid
+            container
+            justifyContent={{ md: 'flex-start', sm: 'center' }}
+            display="flex"
+            sx={{ ml: -1, mt: 3 }}
+          >
+            <Grid item xs={1}>
+              <Tooltip title="GitHub">
+                <IconButton component="a" sx={{ mr: 1 }} href={user.github}>
+                  <GitHubIcon color="primary" />
+                </IconButton>
+              </Tooltip>
+            </Grid>
+            <Grid item xs={1}>
+              <Tooltip title="LinkedIn">
+                <IconButton component="a" sx={{ mr: 1 }} href={user.linkedIn}>
+                  <LinkedInIcon color="primary" />
+                </IconButton>
+              </Tooltip>
+            </Grid>
+            <Grid item xs={1}>
+              <Tooltip title="Porfolio">
```

Comment:
There looks like there's a typo here - "Portfolio"

Reply:
Fixed it. Thank you for pointing this out.